lib/js/app/queryCreator/components/OrderBy/components/OrderByProperty/OrderByProperty.component.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {
+  FC,
+  useState,
+  useRef,
+  useContext,
+  useEffect,
+  useCallback,
+} from 'react';
+import { ActionButton } from '@keen.io/ui-core';
+import { transparentize } from 'polished';
+import { colors } from '@keen.io/colors';
+
+import {
+  Container,
+  DropdownContent,
+  StyledPropertyItem,
+} from './OrderByProperty.styles';
+
+import PropertyGroup, { PropertyItem } from '../../../PropertyGroup';
+import Dropdown from '../../../Dropdown';
+import Property from '../../../Property';
+import EmptySearch from '../../../EmptySearch';
+import PropertiesTree from '../../../PropertiesTree';
+import DirectionList from '../DirectionList';
+
+import text from './text.json';
+
+import { SearchContext } from '../../../../contexts';
+
+import { Direction } from '../../types';
+
+type Props = {
+  /** Properties tree */
+  properties: Record<string, string[] | Record<string, any>>;
+  /** Property */
+  property?: string;
+  /** Direction */
+  direction?: Direction;
```

Comment:
`direction` is not self-descriptive. Could You please rename that property to something more meaningful like for eg. orderDirection ?